WordPress/Classes/ApplicationToken/ApplicationTokenItemView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import DesignSystem
+
+struct ApplicationTokenItemView: View {
+    @State private var isConfirmingDeletion: Bool = false
+
+    private let token: ApplicationTokenItem
+
+    init(token: ApplicationTokenItem) {
+        self.token = token
+    }
+
+    var body: some View {
+        Form {
+            Section {
+                VStack(alignment: .leading) {
+                    Text(Self.name)
+                        .font(.headline)
+                        .foregroundStyle(.secondary)
+                    Text(token.name)
+                }
+                .contextMenu(ContextMenu(menuItems: {
+                    Button(SharedStrings.Button.copy, systemImage: "doc.on.doc.fill") {
+                        UIPasteboard.general.string = token.name
+                    }
+                }))
+            }
+
+            Section(Self.security) {
+                DSListDetailItem(title: Self.creationDate, value: token.createdAt.formatted())
+
+                if let lastUsed = token.lastUsed {
+                    DSListDetailItem(title: Self.lastUsed, value: lastUsed.formatted())
+                }
+
+                if let lastIpAddress = token.lastIpAddress {
+                    DSListDetailItem(title: Self.lastUsedIp, value: lastIpAddress)
+                }
+            }
+        }
+        .navigationTitle(token.name)
+        .navigationBarTitleDisplayMode(.inline)
+    }
+}
+
+// MARK: - Localization
+
+private extension ApplicationTokenItemView {
+    static var name: String { NSLocalizedString("application-password.item.name", value: "Name", comment: "Title of row for displaying an application password name") }
```

Comment:
Keys should use [camel case](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys) (not sure why it says "reverse-DNS").

```suggestion
    static var name: String { NSLocalizedString("applicationPassword.item.name", value: "Name", comment: "Title of row for displaying an application password name") }
```